components/raftstore/src/store/worker/split_controller.rs

```diff
@@ -1563,6 +1681,96 @@ mod tests {
         split_cfg_manager.dispatch(config_change).unwrap();
     }
 
+    #[test]
+    fn test_collect_cpu_stats() {
+        let auto_split_controller = AutoSplitController::default();
+        let region_cpu_map = auto_split_controller.collect_cpu_stats(vec![]);
+        assert!(region_cpu_map.is_empty());
+
+        let ab_key_range_tag = Arc::new(TagInfos {
+            store_id: 0,
+            region_id: 1,
+            peer_id: 0,
+            key_ranges: vec![(b"a".to_vec(), b"b".to_vec())],
```

Comment:
could you add a test case about one tag which has multi ranges? as you said, it exists.

Reply:
Ok, I will add later.

Reply:
Added. PTAL.